build.sbt

```diff
@@ -158,6 +159,18 @@ lazy val contribs = (project in file("contribs"))
     Compile / compile := ((Compile / compile) dependsOn createTargetClassesDir).value
   )
 
+// TODO javastyle tests
+// TODO unidoc
+lazy val storage = (project in file("storage"))
+  .settings (
+    name := "delta-storage",
+    commonSettings,
+    releaseSettings,
+    libraryDependencies ++= Seq(
+      "org.apache.hadoop" % "hadoop-client-api" % "3.3.1" % "provided"
```

Comment:
why this version of hadoop? is there any restriction for this? 
if so put that in comments.